packages/provider/src/RelayClient.ts

```diff
@@ -570,6 +571,9 @@ export class RelayClient {
       }
       config.verifierServerUrl = config.verifierServerUrl ?? DEFAULT_VERIFIER_SERVER_URL
       this.logger.info(`Verifier server API Key is set - setting verifierServerUrl to ${config.verifierServerUrl}`)
+      if (config.paymasterAddress == null) {
```

Comment:
We really need to clean this up. We have both token paymaster and verifying paymaster, and we have different mechanism to check which one to use. We should probably make something generic for all of our paymasters services configurations (currently two, more in the future).